ee/tabby-webserver/src/schema.rs

```diff
@@ -29,6 +36,35 @@ impl Mutation {
         let token = ctx.reset_registration_token().await?;
         Ok(token)
     }
+
+    async fn user_register(
+        ctx: &ServerContext,
+        input: RegisterInput,
+    ) -> FieldResult<RegisterResponse> {
+        let resp = ctx.register(input).await?;
+        Ok(resp)
+    }
+
+    async fn user_login(ctx: &ServerContext, input: LoginInput) -> FieldResult<LoginResponse> {
+        let resp = ctx.login(input).await?;
+        Ok(resp)
+    }
+
+    async fn refresh_token(
+        ctx: &ServerContext,
+        refresh_token: String,
+    ) -> FieldResult<RefreshTokenResponse> {
+        let resp = ctx.refresh_token(refresh_token).await?;
+        Ok(resp)
+    }
+
+    async fn verify_token(
+        ctx: &ServerContext,
+        access_token: String,
```

Comment:
```suggestion
        token: String,
```

Keept it consistent with django